packages/sdk/src/fetcher/serializer/index.ts

```diff
@@ -0,0 +1,15 @@
+import { KPIToken } from "../../entities/kpi-token";
+
+export type KPITokensObject = Record<string, KPIToken>;
```

Comment:
A bit unneeded to me to have this instead of directly writing `Record<string, KPIToken>`, but not anything that is blocking at all.